Assets/Scripts/KuriPhraseManager.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEditor;
+using UnityEngine;
+using Newtonsoft.Json;
+
+namespace MoveToCode {
+    public class KuriPhraseManager : Singleton<KuriPhraseManager> {
+        private List<KuriPhrase> phrases; 
+        private List<KuriPhrase> preloadEncouragement;
+        private List<KuriPhrase> preloadCongratulation;
+        private AWSPollyGetter apg;
+        private int idCount;
+        public bool onlineMode = true;
+
+
+        [HideInInspector] public AudioSource audioSource;
+
+        private KuriPhrase empty = new KuriPhrase(-1, "", KuriPhrase.USECASE.Other, "emptyfile");
+        
+        public KuriPhrase GetPhrase(string lyric) {
+            foreach (KuriPhrase kp in phrases) {
+                if(kp.lyric.Equals(lyric)) {
+                    return kp;
+                }
+            }
+            //if phrase does not exist yet

```

Comment:
shouldn't need this comment if you refactor the name/getter function for `isWorking`